tests/benchmark/test_bloatnet.py

```diff
@@ -0,0 +1,618 @@
+"""
+abstract: Tests [BloatNet](https://bloatnet.info)
+    Test cases for [BloatNet](https://bloatnet.info).
+"""
+
+import pytest
+
+from ethereum_test_base_types import HashInt
+from ethereum_test_forks import Fork
+from ethereum_test_tools import (
+    Account,
+    Alloc,
+    Block,
+    BlockchainTestFiller,
+    Bytecode,
+    Hash,
+    Storage,
+    Transaction,
+    While,
+    compute_create2_address,
+    keccak256,
+)
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+REFERENCE_SPEC_GIT_PATH = "DUMMY/eip-DUMMY.md"
+REFERENCE_SPEC_VERSION = "0.1"
+
+# Constants for CREATE2 address calculation
+CREATE2_MEMORY_OFFSET = 0x1000  # Offset for CREATE2 calculation to avoid collision
+
+
+def generate_create2_address_calculation(
+    factory_address,
+    init_code_hash: bytes,
+) -> Bytecode:
+    """Generate EVM bytecode to calculate a CREATE2 address with salt on stack."""
+    code = Bytecode()
+
+    # Memory layout at CREATE2_MEMORY_OFFSET:
+    # [0xFF][factory_address][salt][init_code_hash]
+    base = CREATE2_MEMORY_OFFSET
+
+    # Store 0xFF prefix at memory[base]
+    code += Op.PUSH1(0xFF) + Op.PUSH2(base) + Op.MSTORE8
+
+    # Store factory address at memory[base+1:base+21]
+    # Handle both string and Address types
+    if isinstance(factory_address, str):
+        addr_hex = factory_address[2:] if factory_address.startswith("0x") else factory_address
+        factory_bytes = bytes.fromhex(addr_hex)
+    else:
+        # Assume it's an Address object with bytes representation
+        factory_bytes = bytes(factory_address)
+    code += Op.PUSH20(int.from_bytes(factory_bytes, "big")) + Op.PUSH2(base + 1) + Op.MSTORE
+
+    # Store salt at memory[base+21:base+53] (base+0x15)
+    # Assumes salt is already on stack
+    code += Op.PUSH2(base + 0x15) + Op.MSTORE
+
+    # Store init code hash at memory[base+53:base+85] (base+0x35)
+    code += Op.PUSH32(int.from_bytes(init_code_hash, "big")) + Op.PUSH2(base + 0x35) + Op.MSTORE
+
+    # Calculate keccak256 of 85 bytes starting at memory[base]
+    code += Op.PUSH1(0x55) + Op.PUSH2(base) + Op.SHA3
+
+    # The address is the last 20 bytes (already on stack)
+    return code
+
+
+@pytest.mark.valid_from("Prague")
+@pytest.mark.parametrize("final_storage_value", [0x02 << 248, 0x02])
```

Comment:
I believe both these params will yield the same test. If you want to test storing 0x0200..00 and also storing 0x0000..02 then you likely want to use `Hash(0x02)`, this will left-pad the data (so you get `0x00..02` (32 bytes)) and this will then yield the 0x02 on stack and not the 0x02..00.